.github/workflows/linux.yml

```diff
@@ -102,7 +111,7 @@ jobs:
           export LD_LIBRARY_PATH="${{ env.CMAKE_BUILD_DIR }}/vcpkg_installed/x64-linux/lib/":${LD_LIBRARY_PATH}
           cmake --build  "${{ env.CMAKE_BUILD_DIR }}" --target bundle --config ${{ env.BUILD_TYPE }}
           echo "ARTIFACT_PATHNAME=${{ env.CMAKE_BUILD_DIR }}/QField-x86_64.AppImage" >> $GITHUB_ENV
-          echo "ARTIFACT_NAME=qfield-${{ env.CI_PACKAGE_FILE_SUFFIX }}-linux-x64.AppImage" >> $GITHUB_ENV
+          echo "ARTIFACT_NAME=${{ env.CI_PACKAGE_FILE_BASENAME }}-linux}-${{ matrix.qt_version_suffix}}-x64.AppImage" >> $GITHUB_ENV
```

Comment:
Are you sure about this?

Reply:
```suggestion
          echo "ARTIFACT_NAME=qfield-${{ env.CI_PACKAGE_FILE_SUFFIX }}-${{ matrix.qt_version_suffix}}-x64.AppImage" >> $GITHUB_ENV
```

better?

Reply:
Yep that'll do. I will let you merge and keep my hands off this PR :)